src/components/Editor/SearchBar.js

```diff
@@ -378,6 +379,27 @@ class SearchBar extends Component {
     );
   }
 
+  renderSearchType() {
+    return dom.div(
+      { className: "search-type-toggles" },
+      dom.span(
+        {
+          className: "label"
+        },
+        "Search:"
+      ),
+      dom.span(
+        {
+          className: "search-type-name",
+          onClick: () => {
+            this.props.setActiveSearch("symbol");
+          }
+        },
+        "functions"
```

Comment:
Lets change this to `Search for functions` and move it to L10N
https://github.com/devtools-html/debugger.html/blob/master/docs/local-development.md#l10n